test/e2e/sagemaker/bootstrap_resources.py

```diff
@@ -22,13 +22,13 @@
 
 @dataclass
 class TestBootstrapResources:
-    DataBucketName: str
     ExecutionRoleARN: str
+    DataBucketName: str
```

Comment:
Was this an automatic formatter change?

Reply:
Actually, this is a fix for the race condition involving role creation and usage for integration tests. I missed porting over the change in service_bootstrap.py because I only ran local Pytest as the end to end integration tests are not working. Thanks for catching this. I fixed it below.